security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainAdd.java

```diff
@@ -257,12 +257,12 @@ private boolean processAudit(OperationContext context, String securityDomain, Mo
         AuditInfo auditInfo = new AuditInfo(securityDomain);
         for (Property moduleProperty : node.asPropertyList()) {
             ModelNode module = moduleProperty.getValue();
-            String codeName = LoginModuleResourceDefinition.CODE.resolveModelAttribute(context, module).asString();
+            String codeName = ProviderModuleResourceDefinition.CODE.resolveModelAttribute(context, module).asString();
```

Comment:
`ProviderModuleResourceDefinition.CODE` is the same instance as `LoginModuleResourceDefinition.CODE` - see https://github.com/wildfly/wildfly/pull/9517/commits/cccccda652bca4e07f00d00a98b960e05b283264#diff-2ebc701847b6432addeb88287a919a21R40 . Hence the line 260 would work well also unchanged. I changed it so that the people that come after me do not need to wonder why audit is referring to login module, even if there is no direct relationship between audit and login module in the management model.